guides/06_Meeting_Events.md

```diff
@@ -102,6 +102,8 @@ The Chime SDK for JavaScript sends these meeting events.
 |`videoInputFailed`      |The camera selection failed.
 |`signalingDropped`      |The WebSocket failed or closed with an error.
 |`receivingAudioDropped` |A significant number of receive-audio packets dropped.
+|`sendingAudioFailed`    |Audio packets are not being sent out for a considerable timespan
```

Comment:
nit: End with a dot.

Also, would it be better for builders to understand if we link to a documentation or add explicit details on configuring the timespan here over "considerable timespan"?

Reply:
Please correct me if we anywhere document what the builders are supposed to do if they receive this event? If not, we can add more here as well as add an use case under README for surfacing it more.